tests/core/utilities/test_math.py

```diff
@@ -1,16 +1,15 @@
 import pytest
 
-from hypothesis import (
-    given,
-    strategies as st,
-)
-
 from web3._utils.math import (
     percentile,
 )
 from web3.exceptions import (
     InsufficientData,
 )
+from web3.tools.hypothesis import (
```

Comment:
This import looks broken.

Reply:
I renamed this to be just hypothesis and import from there as the two imports are `given` and `strategies` which you said elsewhere should only be imported from hypothesis